packages/cactus-plugin-satp-hermes/package.json

```diff
@@ -65,6 +65,12 @@
     "codegen:openapi": "npm run generate-sdk",
     "codegen:proto": "npm run generate-proto",
     "codegen:abi": "yarn forge:all && abi-types-generator './src/solidity/generated/satp-wrapper.sol/SATPWrapperContract.json' --output='./src/main/typescript/generated'",
+    "db:init": "run-s db:init:local db:init:remote",
```

Comment:
this looks ok, but I thought there were some changes on these commands. make sure you have the latest rebased version of satp-dev